source/administration/config-settings.rst

```diff
@@ -3108,6 +3131,22 @@ Prefix on the Elasticsearch index name. Enables the use of Mattermost Elasticsea
 | This feature's ``config.json`` setting is ``"IndexPrefix": ""`` with string input      |
 +----------------------------------------------------------------------------------------+
 
+Live Indexing Batch Size
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+XXXXXX
```

Comment:
@grundleborg please help with describing this setting

Reply:
This setting controls how many new posts are batched together before they are added to the Elasticsearch Index. If it is set to `1`, all new posts are indexed immediately. If it is set to a number greater than 1, then they are batched together until that number of new posts is reached, and then they are all indexed in a batch. On most installs, it should be left as the default of `1` but on some very large installs (think multiple messages per second on average), it may be necessary to increase this value to avoid hitting the rate limit of your Elasticsearch cluster.